CorsixTH/Lua/date.lua

```diff
@@ -48,6 +48,22 @@ local month_length = {
 
 local hours_per_day = 50
 
+-- Number of days in the year before the first of each month
+local cumulative_days = {
+  0,
+  31,
+  59,
+  90,
+  120,
+  151,
+  181,
+  212,
+  243,
+  273,
+  304,
+  334,
+}
```

Comment:
Don't add magic numbers that need to be kept in sync with other magic numbers.

Compute this table from the table above.